### PR TITLE
fix(slider): make disabled state look like mocks

### PR DESCRIPTION
**Pull request by mmalerba**
part of #2603 making slider look like mocks for all states

### CODE REVIEW

File: src/lib/slider/slider.ts [hunk 1]
```diff
@@ -245,11 +246,24 @@ export class MdSlider implements ControlValueAccessor {
     return (this.direction == 'rtl' && !this.vertical) ? !this.invertAxis : this.invertAxis;
   }
 
+  get thumbGap() {
```
Comment (mmalerba) [inline src/lib/slider/slider.ts]: Done

File: src/lib/slider/slider.ts [hunk 2]
```diff
@@ -245,11 +246,29 @@ export class MdSlider implements ControlValueAccessor {
     return (this.direction == 'rtl' && !this.vertical) ? !this.invertAxis : this.invertAxis;
   }
 
+  /**
+   * The amount of space to leave between the slider thumb and the track fill & track background
+   * elements.
+   */
+  get thumbGap() {
+    return this.disabled ? 7 : 0;
```
Comment (jelbourn) [inline src/lib/slider/slider.ts]: Move `7` into a constant?

File: src/lib/slider/slider.ts [hunk 3]
```diff
@@ -245,11 +246,29 @@ export class MdSlider implements ControlValueAccessor {
     return (this.direction == 'rtl' && !this.vertical) ? !this.invertAxis : this.invertAxis;
   }
 
+  /**
+   * The amount of space to leave between the slider thumb and the track fill & track background
+   * elements.
+   */
+  get thumbGap() {
```
Comment (jelbourn) [inline src/lib/slider/slider.ts]: This looks like it can be `private`

**Comment by mmalerba** [comment]
Done

**Pull request merged**